### DISCUSSION TITLE
How to give a Button a Border?

### DISCUSSION THREAD

**Discussion by CrawfishPress** [emilk/egui#1924] [Q&A]
Either a Border, or some indication that it's been pressed - trying to use buttons for Tabs, want a visual indicator of which is selected. (Like radio-buttons). I thought giving them a Frame might work:
```
            ui.horizontal(|ui| {
                    ui.visuals_mut().button_frame = true;
                    let load_some_button = egui::Button::new(tab_load_screen_txt).frame(true);
                    if ui.add(load_some_button).clicked() { //... }
```
but apparently not - or I'm doing it wrong. :)

Appreciate any help.

**Comment by emilk** [comment]
I think what you are looking for is https://docs.rs/egui/latest/egui/widgets/struct.SelectableLabel.html
